### DISCUSSION TITLE
gauge chart should allow in customize to configure legend position and alignment

### DISCUSSION THREAD

**Discussion by nigzak** [apache/superset#27667] [General]
## Screenshot

![image](https://github.com/apache/superset/assets/102737855/841bbfa0-7f65-4aaa-86b6-946d56b1a55d)

## Description

Gauge Chart: let the user decide to position the legend

## Design input
the current legend is INSIDE the gauge which leads that depending the size the legend is partial not visible (depending how many values are there)
In my screenshot there are 6 values - but it is only showing 4 - and it is already smallest font style selected

options which should be possible:

- move legend LEFT to the gauge as list
- move legend RIGHT to the gauge as list
- hide legend at all
- current way (inside gauge)
- top of gauge
- botton of gauge

maybe suboption for LEFT/RIGHT: center, top or bottom aligned
TOP/BOTTOM might be similar as other charts (ex: line-chart) with extra buttons if it is more that you can click if it is too many to show them

![image](https://github.com/apache/superset/assets/102737855/9e7085eb-45df-4ad8-a524-71d85049ecbd)

**Comment by michael-s-molina** [comment]
Moving to a GitHub discussion as this is more of a feature request.

**Comment by nigzak** [comment]
seems nobody is interested here -> closing

**Comment by tanmay-kulkarni** [comment]
I'd love to have this feature too! it'd be nice to have this reopened.

**Comment by bsenthilbalu** [comment]
Love to have this feature. Particularly when we have multiple metrics, the pointer needle overlaps and is not very clear to read.
